CMakeLists.txt

```diff
@@ -136,6 +137,10 @@ mark_as_advanced(WarpX_MPI_THREAD_MULTIPLE)
 
 option(WarpX_amrex_internal                    "Download & build AMReX" ON)
 
+if(WarpX_HEFFTE AND NOT WarpX_MPI)
+    message(FATAL_ERROR "WarpX_HEFFTE (${WarpX_HEFFTE}) can only be used if WarpX_MPI is ON.")
+endif()
```

Comment:
I discussed this with @RemiLehe and this should not be needed.
We can use the IGF solver for multiple nodes without having single-node FFT support.

We can certainly enforce this if it makes things clearer for people, but it might come with a downside: if one turns FFT on in RZ, this would trigger more dependencies (BLAS++ and LAPACK++). Thus, I would leave like this for now.